src/pages/Staking/DelegateStakeTable.tsx

```diff
@@ -242,22 +242,34 @@ const DelegateStake = () => {
       id: 'action',
       header: '',
       cell: ({ row }) => {
-        return (
-          <div className="pr-6">
-            <Button
-              buttonType={ButtonType.PRIMARY}
-              active={true}
-              title="Manage Stake"
-              text="Stake"
-              onClick={(e) => {
-                e.stopPropagation();
-                if (walletAddress) {
+        const ownGateway = row.original.owner === walletAddress?.toString();
+        const btn = (
+          <Button
+            buttonType={ButtonType.PRIMARY}
+            active={true}
+            title="Manage Stake"
+            text="Stake"
+            onClick={(e) => {
+              e.stopPropagation();
+              if (walletAddress) {
+                if (!ownGateway) {
                   setStakingModalWalletAddress(row.getValue('owner') as string);
-                } else {
-                  setIsConnectModalOpen(true);
                 }
-              }}
-            />
+              } else {
+                setIsConnectModalOpen(true);
+              }
+            }}
+          />
+        );
+        return (
+          <div className="pr-6">
+            {ownGateway ? (
+              <Tooltip message="Delegate staking is not supported for gateways you operate. Please use operator staking on your gateway details page.">
+                <div className="pointer-events-none opacity-30">{btn}</div>
```

Comment:
![image](https://github.com/user-attachments/assets/bfdc18a4-85e9-4d8f-8f36-4a08ae38e0aa)
This is causing some difference in sizing on the button having it wrapped in the div i think

Reply:
Thanks for catching this; pushed a fix to Tooltip.tsx to remove an unnecessary div that was causing problems.